config/crd/bases/ocs.openshift.io_autostoragescalings.yaml

```diff
@@ -0,0 +1,127 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.16.1
+  name: autostoragescalings.ocs.openshift.io
+spec:
+  group: ocs.openshift.io
+  names:
+    kind: AutoStorageScaling
```

Comment:
This file (and a few others) look like they need to be removed from the commit, generated from a previous type name.

Reply:
thanks for catching it out